src/services/ipfs-service.ts

```diff
@@ -6,13 +6,17 @@ import { logger } from '../logger/index.js'
 import * as dagJose from 'dag-jose'
 import type { IPFS } from 'ipfs-core-types'
 import { inject, singleton } from 'tsyringe'
-import { toCID } from '@ceramicnetwork/common'
+import { AnchorCommit, toCID } from '@ceramicnetwork/common'
+import { serialize, MsgType } from '@ceramicnetwork/core/lib/pubsub/pubsub-message'
```

Comment:
FYI: I will update this to use `'@ceramicnetwork/core/pubsub/pubsub-message'` which is exported from the release candidate version of `core`, but I am currently having trouble with the named exports from `core`